config/config.json

```diff
@@ -3,17 +3,9 @@
   "name": "Chronicle Backstory",
   "acronym": "CHBS",
   "description": "Chronicle Backstory combines intelligence about global threats in the wild, threats inside your network, and unique signals about both.",
-  "entityTypes": [
-    "IPv4",
-    "IPv6",
-    "domain",
-    "MAC",
-    "hash"
-  ],
+  "entityTypes": ["IPv4", "IPv6", "domain", "MAC", "MD5", "SHA1", "SHA256"],
```

Comment:
The `MAC` type isn't actually a supported entity type.  Can you remove that one?